rockgame.c

```diff
@@ -0,0 +1,83 @@
+//Olivier Gabison - Rock Paper Scissors Game. I've been staring at my code and I'm unsure why it
+//loops through the whole program after 1 input?
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main(){
+
+  int wins_human = 0;
+  int wins_comp = 0;
+
+  for(int i = 0; i < 10; i++){
+
+    if(wins_comp > 2){
+      printf("The computer wins!\n");
+      break;
+    } else if (wins_human > 2){
+      printf("The human wins!\n");
+      break;
+    }
+
+    printf("Current Score - You: %d Computer: %d\n", wins_human, wins_comp);
+    printf("Please choose either ROCK [R], PAPER [P], or Scissors[S]\n");
+    char answer;
+    scanf("\n%d",&answer);
```

Comment:
hey olivier! your code actually only needed one modification for it to work perfectly! you see the %d in the scanf there? It should be a c since you're accepting a character as input! Once you fix that, your program compiles without warning and runs perfectly!